.github/workflows/continuous-integration-workflow.yml

```diff
@@ -19,9 +19,10 @@ env:
   ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
 
 jobs:
-  initiate_error_tracking:
-    name: Initiate job-level error tracking
+  parent:
+    name: Pre-requisites for building
     runs-on: ubuntu-latest
+    if: github.repository == 'spring-projects/spring-security'
```

Comment:
Is there a way for this to be done through the environment so users can activate checks in their fork without changing the source?

Reply:
Hypothetically, `prerequisites` could emit some kind of job outcome that the other tasks could check for. For example, they could possibly check for the existence of the `job-initiate-error-tracking.txt` file.